zubbi/scraper/connections/gerrit.py

```diff
@@ -41,3 +88,10 @@ def init(self):
     @property
     def provider(self):
         return "gerrit"
+
+    def get_web_url_builder(self, web_type, web_url, url):
+        web_url = web_url or url
+        url_builder_class = self.WEB_URL_BUILDERS.get(web_type)
+        if url_builder_class is not None:
+            return url_builder_class(web_url)
+        LOGGER.error("Unsupported web_type: ''", web_type)
```

Comment:
How about throwing an exception here instead? Otherwise this will happen during runtime when `build_*_url()` is called on `None`.